src/icons/fileIcons.ts

```diff
@@ -1473,5 +1473,15 @@ export const fileIcons: FileIcons = {
       fileNames: ['jsconfig.json'],
       fileExtensions: ['jsconfig.json'],
     },
+    {
+      name: 'ember',
+      fileNames: [
+          '.ember-cli',
+          '.ember-cli.js',
+          'ember-cli-builds.js',
+          'fastboot.js',
+      ],
+      fileExtensions: ['local.js']
```

Comment:
I do not see this file extension as specific for ember. Please remove it because it will be confusing to people who are having `*.local.js`-files in their projects.

Reply:
Alright, will do